README.md

```diff
@@ -409,56 +409,36 @@ Components](https://github.com/styled-components/styled-components) and
 Purview components do not run in the browser.
 
 ## Error handling
-Uncaught errors within Purview will bubble up as
-[`uncaughtException`][uncaught-exception] or
-[`unhandledRejection`][unhandled-rejection] events. These can be caught and
-handled by registering event listeners on `process`.
+Normally, when an error occurs in the context of a Purview component, it will 
+appear as top-level [`uncaughtException`][uncaught-exception] or
+[`unhandledRejection`][unhandled-rejection] events. Purview provides the option
+to observe or modify these errors with a custom error handler before they
+appear within these top-level events.
 
-Additionally, certain classes of errors can be instrumented in Purview prior to
-going to these top-level events:
+In order to add a custom error handler, pass an `onError` function as an option
+to `Purview.render()`. When provided, the handler is invoked for these classes
+of Purview errors:
 
 - Errors within event callbacks.
 - Errors in the component render path.
 - Errors when setting initial component state, after WebSocket connection is
 established.
 
-Register an instrumentation function by passing it in as an option parameter to
-`Purview.render()`:
+If your handler does not itself throw an error, Purview will re-throw the error
+in order to prevent execution of further code.
 
 ```tsx
-const app = express()
-
 app.get("/", async (req, res) => {
   function onError(error: unknown): void {
-    // Instrument your error here, such as sending it to a monitoring service
-    // or adding context from the request.
+    // Instrument the error here, such as to add context:
+    error.userID = req.userID
+    // Or send it to a monitoring service:
+    logError(error)
+    // Or throw a custom error class:
+    throw new PurviewCustomError(error)
   }
 
-  // Pass onError inside Purview.render's third argument.
   const appHTML = Purview.render(<App />, req, { onError })
```

Comment:
could we add a comment after this to indicate that there would likely be other code in the route handler?

 `// ...`

Reply:
Thanks for calling this out. I agree it is confusing without the comment, so it's been added in the latest commit.